Stratis.Bitcoin.Features.Consensus/PowConsensusValidator.cs

```diff
@@ -167,20 +166,28 @@ public virtual void ExecuteBlock(ContextInformation context, TaskScheduler taskS
             DeploymentFlags flags = context.Flags;
             UnspentOutputSet view = context.Set;
 
+            int lastCheckpointHeight = this.Checkpoints.GetLastCheckpointHeight();
```

Comment:
I had in mind that this info will go in the Context and so be accessible every where.
But I guess this is fine like this.